### PR TITLE
Update cloudflare-workers.md to add missing import

### PR DESCRIPTION
**Pull request by kleenkanteen**

### CODE REVIEW

File: getting-started/cloudflare-workers.md [hunk 1]
```diff
@@ -273,6 +273,14 @@ app.put('/upload/:key', async (c, next) => {
 This is the only case for Module Worker mode.
 If you want to use Variables or Secret Variables in Middleware, for example, "username" or "password" in Basic Authentication Middleware, you need to write like the following.
 
+First add this import at the top of `index.tsx`:
```
Comment (yusukebe) [inline getting-started/cloudflare-workers.md:276]: The description is not needed. Just write the code like the following:

```ts
import { basicAuth } from 'hono/basic-auth'

// ...

app.use('/auth/*', async (c, next) => {
  const auth = basicAuth({
    username: c.env.USERNAME,
    password: c.env.PASSWORD,
  })
  return auth(c, next)
})
```

**Comment by yusukebe** [comment]
Hi @kleenkanteen 

Thanks. But, the description is not needed. We can add the import statement to the example code.

File: getting-started/cloudflare-workers.md (hunk 1 above)
Comment (kleenkanteen) [reply getting-started/cloudflare-workers.md:276]: fixed

File: getting-started/cloudflare-workers.md [hunk 2]
```diff
@@ -273,7 +273,13 @@ app.put('/upload/:key', async (c, next) => {
...
```
Comment (yusukebe) [inline getting-started/cloudflare-workers.md]: Please delete this line.

Comment (kleenkanteen) [reply getting-started/cloudflare-workers.md]: done

**Comment by yusukebe** [comment]
@kleenkanteen 

Thanks! I'll merge it now.

**Pull request merged**